cmd/gitlab.go

```diff
@@ -9,303 +9,77 @@ import (
 	"time"
 
 	"github.com/google/go-cmp/cmp"
-	"github.com/prometheus/client_golang/prometheus"
-	"github.com/xanzy/go-gitlab"
-
 	log "github.com/sirupsen/logrus"
+	"github.com/xanzy/go-gitlab"
+	"go.uber.org/ratelimit"
 )
 
-func projectExists(p Project) bool {
-	for _, cp := range cfg.Projects {
-		if cmp.Equal(p, cp) {
-			return true
-		}
-	}
-	return false
-}
+var statusesList = []string{"running", "pending", "success", "failed", "canceled", "skipped", "manual"}
```

Comment:
```suggestion
var statusesList = [...]string{"running", "pending", "success", "failed", "canceled", "skipped", "manual"}
```

as this slice will not expand further

Reply:
👍

Reply:
this actually breaks the function using it... I'd keep the slice that is more flexible

Reply:
ok 👍